templates/loop.twig

```diff
@@ -1,41 +1,53 @@
-{% for post in posts %}
-
-  <section class="post-wrapper">
-
-    <article class="post-{{ post.id }} {{ post.class }}">
-      <header>
-        {% if post.thumbnail %}
-          <!-- Thumbnail -->
-          <div class="thumbnail">
-            <img src="{{post.thumbnail.src | resize(120, 120, 'bottom')}}" class="my-thumb-class" alt="Image for {{post.title}}" />
-          </div>
-        {% endif %}
-
-        <!-- Display -->
-        <h2 class="post-title"><a href="{{ post.link }}">{{ post.title }}</a></h2>
-        <div class="meta">
-          <div class="date">
-            <span>{{ function( 'the_time', 'F j, Y')}} </span>
-            <span>{{ function( 'the_time', 'g:i a')}} </span>
-          </div>
-          <div class="author">{{ __('Published by', 'gesso') }} <a href="{{ post.author.link }}">{{ post.author }} </a></div>
-          <div class="comments">
-            {% if post.comments %}
-              <span>Comments: {{ post.comments | length }}</span>
+<div class="post-loop">
+
+    {% for post in posts %}
+
+        <article class="article post-{{ post.id }} {{ post.class }}">
```

Comment:
it's confusing to have all three of these classes.  what does 'post.class' resolve to?  just 'post'?